channels/generic/websocket.py

```diff
@@ -79,14 +83,16 @@ def send(self, text_data=None, bytes_data=None, close=False):
         if close:
             self.close(close)
 
-    def close(self, code=None):
+    def close(self, code=None, reason=None):
         """
         Closes the WebSocket from the server end
         """
+        message = {"type": "websocket.close"}
         if code is not None and code is not True:
```

Comment:
For my curiosity, why is there a check for the "code not being True"?

Reply:
Oh, hard questions!   I think this is a compatibility check.  It's been a while, but IIRC, there was a case where a "True" value was being passed to close...  I need to look.

Reply:
Oh, this is not my code.  Phew.  So, your guess is as good as mine.  Maybe in the distant past, someone would call `close(True)` for whatever reason and this is trying to allow for that?  Nothing in the library or unit tests which is doing that, though.